ci-operator/jobs/openshift/router/openshift-router-master-presubmits.yaml

```diff
@@ -490,6 +490,81 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )images,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - ^master$
+    - ^master-
+    cluster: build05
+    context: ci/prow/perfscale-aws-ingress-perf
+    decorate: true
+    decoration_config:
+      timeout: 5h0m0s
+    labels:
+      ci-operator.openshift.io/cloud: aws
+      ci-operator.openshift.io/cloud-cluster-profile: aws-perfscale
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-router-master-perfscale-aws-ingress-perf
```

Comment:
Perhaps this name needs to match what is being reported by CI in https://github.com/openshift/release/pull/43459#issuecomment-1724511360?

Reply:
It appears the test ran successfully yesterday after i issued the `/retest`